en/support/index.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: page
+title: Version Support
+menu: support
+lang: en
+---
+
+# Version Support
+
+Only the latest version of any given major release line is supported.
+
+Versions that are EOL (end-of-life) _may_ receive updates for critical security vulnerabilities, but the Express team offers no guarantee and does not plan to address or release fixes for any issues found.
+
+| Major Version | Support Start Date | Support End Date |
+| -- | -- | -- |
+| [**v5.x**{: style="color: gray; opacity: 0.8" }](/{{page.lang}}/5x/api.html) | **not yet released**{: style="color: gray; opacity: 0.8" } | **not yet released**{: style="color: gray; opacity: 0.8" } |
```

Comment:
color does not meet contrast requirements for a11y in dark mode

Reply:
what color would you suggest that's lighter than the rest?

Reply:
For dark mode, I would recommend this color #abc8c3. Additionally, I think it would be better to manage the styles in the CSS file ([`styles.css`](https://github.com/expressjs/expressjs.com/blob/gh-pages/css/style.css))

Reply:
You can add a class in the same way as a style, but in this case with `{: .my-class }`, just as I suggested in the changes.

```suggestion
| [**v5.x**{: .no-release }](/{{page.lang}}/5x/api.html) | **not yet released**{: .no-release } | **not yet released**{: .no-release } |
```

Reply:
done, thanks!